runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -560,9 +561,18 @@ private void hop(boolean previous)
 
 			world = worlds.get(worldIdx);
 
-			// Check world region if filter is enabled
-			if (config.quickHopRegionFilter() != RegionFilterMode.NONE && world.getRegion() != config.quickHopRegionFilter().getRegion())
+			// Check if quick-hop is disabled.
+			if (!config.quickHopAURegion() && !config.quickHopDERegion()
+					&& !config.quickHopGBRegion() && !config.quickHopUSRegion())
 			{
+				return;
+			} // Check world region if filters are enabled
+			else if (!(config.quickHopAURegion() && world.getRegion() == WorldRegion.AUSTRALIA) &&
+					!(config.quickHopDERegion() && world.getRegion() == WorldRegion.GERMANY) &&
+					!(config.quickHopUSRegion() && world.getRegion() == WorldRegion.UNITED_STATES_OF_AMERICA) &&
+					!(config.quickHopGBRegion() && world.getRegion() == WorldRegion.UNITED_KINGDOM))
```

Comment:
Our preference of style for multiline conditionals is to start lines with boolean operators:
```suggestion
			else if (!(config.quickHopAURegion() && world.getRegion() == WorldRegion.AUSTRALIA)
				&& !(config.quickHopDERegion() && world.getRegion() == WorldRegion.GERMANY)
				&& !(config.quickHopUSRegion() && world.getRegion() == WorldRegion.UNITED_STATES_OF_AMERICA)
				&& !(config.quickHopGBRegion() && world.getRegion() == WorldRegion.UNITED_KINGDOM))
```